### DISCUSSION TITLE
Displaying subcategories in category listing admin page

### DISCUSSION THREAD

**Discussion by BrainforgeUK** [joomla/joomla-cms#47096] [2. Cold Features]
### Steps to reproduce the issue

Create a heirarchy of article categories
cat1
cat1/cat1.1
cat1/cat1.2
cat1/cat1.3
cat1/cat1.4/cat1.4.1

In the article category listing search for cat1 using the id:8 search criteria as in attached screenshot.

![screen shot 2022-11-25 at 20 54 23](https://issues.joomla.org/uploads/1/609bb6d47348e66df7f130a07df57ef0.png)

### Expected result

Expect to see cat1.1, cat1.2, cat1.3 and cat1.4 listed below cat1 and /or the presence of subcategories indicated by a button count like the number of (un)published article buttons.

### Actual result

Subcategories not displayed, user is left wondering have I lost them? The max levels filter option does nothing, only adding to the confusion.

### System information (as much as possible)

PHP 8.1

### Additional comments

I suggest this be tackled in separate steps, the easiest being first.
(a) Required for step (b).
Changed the function of the ID:x search criteria so that a comma separated list of IDs can be entered i.e. ID:8,4,6,2,9 All the IDs specified will be returned.
(b) Add a column of subcategory count buttons (before?) next to the (un)published etc article button counts. Clicking this will then drill down showing all subcategories of the current category. This will use the new IDx,y,z comma search criteria.
(c) In the row above the table (where 9/9 is displayed), on left hand side show the parent category (or breadcrumb category tree?).
(d) Modify the display so that it shows the category tree starting from the selected category. i.e. such as displayed when there are no filter criteria ... or would it be better to do this first when a category is search for using ID:x?

This issue arose with another component with links to com_categories for article categories referred to in / used by that other component.

**Comment by ot2sen** [comment]
Why would you expect non-related subcategories to be shown on an extremely specific ID search. It does what you ask for, doesn´t it?

**Comment by BrainforgeUK** [comment]
True it does what asked for ... BUT there are **related** subcategories which are not apparent to the user, that is the issue I am raising. The ID:x,y,z search criteria feature would be the first step to resolving that, the others provide a more complete (and preferred) solution.

**Comment by simbus82** [comment]
> Expect to see cat1.1, cat1.2, cat1.3 and cat1.4 listed below cat1.

Great UX enhancement!

> Why would you expect non-**related** subcategories to be shown on an extremely specific ID search. It does what you ask for, doesn´t it?

it's really logic for who manage contents. A sub-cat it's a part of a category in content management processes.
